src/main/java/com/socrata/api/DatasetDestination.java

```diff
@@ -0,0 +1,5 @@
+package com.socrata.api;
+
+public enum DatasetDestination {
+    OBE, NBE;
```

Comment:
Is the `OBE` here just for the sake of completeness, since the code seems to represent OBE as null to keep the existing semantics unchanged?

Reply:
No, the whole point of this change is to allow targeting the OBE specifically; before your options were "NBE" or "whatever is the default for the domain".

Reply:
It only keeps "!nbe" as "default" in the preexisting codepath.

Reply:
oh ok. Just didn't see DatasetDestination.OBE used anywhere

Reply:
It's not; it's there for user code (specifically, for DataSync) to use.